sense-android-library/documentation/sensor_profiles.md

```diff
@@ -0,0 +1,472 @@
+      "type": "number"
+    },
+    "metadata": {
+      "type": "object",
+      "properties": {
+        "core version": {
+          "description" : "The version of the cortex core",
+          "type": "string"
+        },
+        "module version": {
+          "description" : "The version of the sleep_time module",
+          "type": "string"
+        },
+        "status": {
+          "description" : "The current status of the sleep_time module for debugging module, e.g. awake: too much noise",
+          "type": "string"
+        }
+      }
+    },
+    "hours": {
+      "description" : "The number of actual sleep hours",
+      "type": "integer"
+    },
+    "start_date": {
+      "description" : "The start time of the sleep period in epoch seconds",
+      "type": "integer"
+    }
+  },
+  "required": [
+    "end_time",
+    "hours",
+    "start_time"
+  ]
+}
+~~~
+*changed the sleepTime key to hours, end_date and start_date to end_time and start_time resp.*
+
+### sleep_estimate
+*renamed from sleep_time_estimate*
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "end_time": {
+      "description" : "The end time of the sleep period in epoch seconds",
+      "type": "number"
+    },
+    "history_based": {
+      "description" : "Whether the sleep value is based on the history or on the actual sleep period",
+      "enum": [0, 1]
+    },
+    "metadata": {
+      "type": "object",
+      "properties": {
+        "core version": {
+          "description" : "The version of the cortex core",
+          "type": "string"
+        },
+        "module version": {
+          "description" : "The version of the sleep_time module",
+          "type": "string"
+        },
+        "status": {
+          "description" : "The current status of the sleep_time module for debugging module, e.g. awake: too much noise",
+          "type": "string"
+        }
+      }
+    },
+    "hours": {
+      "description" : "The number of actual sleep hours",
+      "type": "integer"
+    },
+    "start_time": {
+      "description" : "The start time of the sleep period in epoch seconds",
+      "type": "integer"
+    }
+  },
+  "required": [
+    "end_time",
+    "hours",
+    "start_time"
+  ]
+}
+~~~
+*changed the sleepTime key to hours, end_date and start_date to end_time and start_time resp.*
+
+
+
+## Deprecated sensors
+Sensors that output data that will not be stored in the back-end.
+
+Debug ouput of the coaches is stored in sensors but is not needed:
+
+ANDROID / iOS
+* SLEEP_DURATION / sleep_duration_coach
+* MEAL_TIME / meal_time_coach
+* MENTAL_RESILIENCE
+* VISIT_LOCATION
+* PHYSICAL_ACTIVITY / exercise_coach
+* AGORAPHOBIA
+
+Sense library sensors which do not need data in the back-end:
+* linear acceleration
+* linear acceleration (burst-mode)
+* accelerometer (burst-mode)
+* motion energy
+* motion features
+* connection type
+* geofence_sensor_VISIT_LOCATION
```

Comment:
@tedschmidt i'm not sure but i think we still missing locationtrace, physical_activity, step counter, and top location in the deprecated sensors section (that not needed to be stored in backend)